### ISSUE TITLE
Add dropout with argparse flag

### ISSUE THREAD

**Issue by MatthewAlmeida** [MatthewAlmeida/PACS_Sampling_Experiments#4]
Add dropout with flag before the top linear projection layer to pull back on the training fit a little bit.  We want to try longer, slower training with higher regularization to see if we can get some more stable results against the validation set.
